### ISSUE TITLE
ZEO Mismatch

### ISSUE THREAD

**Issue by jamadden** [NextThought/nti.recipes.zodb#17]
Given 
```ini
[zeo]
recipe = nti.recipes.zodb:zeo
storages = users
```

The produced zeo_uris.ini file contains
```ini
[ZODB]
uris = zconfig:////etc/zodb_conf.xml#users
```

But the named database in zodb_conf.xml has a different name:
```
%import zc.zlibstorage

<zodb users_client>
 cache-size 100000
 database-name users_client
 pool-size 60
 <zlibstorage>
     <zeoclient>
       blob-dir /.../users_client.blobs
       name users_client
       server //var/zeosocket
       shared-blob-dir true
       storage 1
     </zeoclient>
   compress false
 </zlibstorage>
</zodb>
```

**Comment by jamadden** [comment]
Something similar affects relstorage in that the case isn't preserved.

**Comment by jamadden** [comment]
Released in 1.0a3.

/cc @jzuech3

**Comment by jzuech3** [comment]
Thanks!
